src/main/kotlin/uk/gov/justice/digital/hmpps/personrecord/service/PersonService.kt

```diff
@@ -60,6 +60,8 @@ class PersonService(
     if (isUpdateEvent(event)) {
       trackEvent(CPR_UPDATE_RECORD_DOES_NOT_EXIST, person)
     }
+    // Use result of search to determine assign of UUID CPR-271
+    searchForRecord(person, searchBySourceSystem = false)
```

Comment:
The boolean here is just passed through into the next method. It might be nicer to have two methods?
Something like
`searchAllSourceSystems`
`searchSameSourceSystem`

We can have a look at this later, easy to refactor and it will work as is.